nucypher/policy/policies.py

```diff
@@ -391,7 +415,8 @@ def enact(self, network_middleware, publish_treasure_map=True) -> dict:
             self.alice.add_active_policy(self)
 
             if publish_treasure_map is True:
-                return self.publish_treasure_map(network_middleware=network_middleware)
+                publication_result = self.inject_alice_publication_threadpool_into(self.publish_treasure_map, network_middleware=network_middleware)
+                # return self.publish_treasure_map(network_middleware=network_middleware)
```

Comment:
we no longer return anything here?

Reply:
Yeah, I've gone back and forth.  What do you think?  At this moment, this function works for my mental model as a void.  If the middleware starts returning a deferred (toward which I'm increasingly leaning), then we will obviously reevaluate.

Reply:
My worry stemmed from the fact that this method (`enact()`) is marked to return a `dict`. The void is fine, but does the signature need to be changed?

Reply:
If this change is accepted, then yes.  (There's lots of that variety of cleanup that will be required on TI; I haven't run mypy on things yet.)